[2025-02-05T14:33:09.032+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-05T14:33:09.050+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.agg_locations_risk_score_run manual__2025-02-04T00:00:00+00:00 [queued]>
[2025-02-05T14:33:09.060+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.agg_locations_risk_score_run manual__2025-02-04T00:00:00+00:00 [queued]>
[2025-02-05T14:33:09.061+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-02-05T14:33:09.073+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtRunLocalOperator): model_credit_fraud.agg_locations_risk_score_run> on 2025-02-04 00:00:00+00:00
[2025-02-05T14:33:09.078+0000] {standard_task_runner.py:63} INFO - Started process 342 to run task
[2025-02-05T14:33:09.081+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_dbt-model_credit_fraud', 'model_credit_fraud.agg_locations_risk_score_run', 'manual__2025-02-04T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/dag_dbt-model_fraud_credit.py', '--cfg-path', '/tmp/tmprrqlymm2']
[2025-02-05T14:33:09.082+0000] {standard_task_runner.py:91} INFO - Job 16: Subtask model_credit_fraud.agg_locations_risk_score_run
[2025-02-05T14:33:09.148+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.agg_locations_risk_score_run manual__2025-02-04T00:00:00+00:00 [running]> on host eb17fa5f9c23
[2025-02-05T14:33:09.248+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_dbt-model_credit_fraud' AIRFLOW_CTX_TASK_ID='model_credit_fraud.agg_locations_risk_score_run' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-04T00:00:00+00:00'
[2025-02-05T14:33:09.251+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-05T14:33:11.812+0000] {local.py:220} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2025-02-05T14:33:11.814+0000] {local.py:436} INFO - Cloning project to writable temp directory /tmp/tmpyyezcadb from /usr/local/airflow/dags/dbt/localiza_dw
[2025-02-05T14:33:11.822+0000] {local.py:447} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/dag_dbt-model_credit_fraud__model_credit_fraud/target/partial_parse.msgpack
[2025-02-05T14:33:11.828+0000] {config.py:348} INFO - Profile caching is enable.
[2025-02-05T14:33:11.836+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2025-02-05T14:33:11.838+0000] {config.py:328} INFO - Profile found in cache using profile: /tmp/cosmos/profile/bdc35343eb23031ee554202b16407e794194afcc614f7ea62bc658c63495a211/profiles.yml.
[2025-02-05T14:33:11.839+0000] {local.py:408} INFO - Trying to run dbtRunner with:
 ['run', '--models', 'agg_locations_risk_score', '--project-dir', '/tmp/tmpyyezcadb', '--profiles-dir', '/tmp/cosmos/profile/bdc35343eb23031ee554202b16407e794194afcc614f7ea62bc658c63495a211', '--profile', '***', '--target', 'prod']
 in /tmp/tmpyyezcadb
[2025-02-05T14:33:11.862+0000] {logging_mixin.py:188} INFO - 14:33:11  Running with dbt=1.6.1
[2025-02-05T14:33:11.991+0000] {logging_mixin.py:188} INFO - 14:33:11  Registered adapter: postgres=1.6.1
[2025-02-05T14:33:12.706+0000] {logging_mixin.py:188} INFO - 14:33:12  Found 4 models, 1 snapshot, 4 seeds, 5 tests, 3 sources, 0 exposures, 0 metrics, 349 macros, 0 groups, 0 semantic models
[2025-02-05T14:33:12.710+0000] {logging_mixin.py:188} INFO - 14:33:12
[2025-02-05T14:33:12.784+0000] {logging_mixin.py:188} INFO - 14:33:12  Concurrency: 1 threads (target='prod')
[2025-02-05T14:33:12.786+0000] {logging_mixin.py:188} INFO - 14:33:12
[2025-02-05T14:33:12.790+0000] {logging_mixin.py:188} INFO - 14:33:12  1 of 1 START sql table model dm_credit_operations.agg_locations_risk_score ..... [RUN]
[2025-02-05T14:33:17.302+0000] {logging_mixin.py:188} INFO - 14:33:17  1 of 1 OK created sql table model dm_credit_operations.agg_locations_risk_score  [SELECT 5 in 4.51s]
[2025-02-05T14:33:17.330+0000] {logging_mixin.py:188} INFO - 14:33:17
[2025-02-05T14:33:17.333+0000] {logging_mixin.py:188} INFO - 14:33:17  Finished running 1 table model in 0 hours 0 minutes and 4.62 seconds (4.62s).
[2025-02-05T14:33:17.351+0000] {logging_mixin.py:188} INFO - 14:33:17
[2025-02-05T14:33:17.353+0000] {logging_mixin.py:188} INFO - 14:33:17  Completed successfully
[2025-02-05T14:33:17.354+0000] {logging_mixin.py:188} INFO - 14:33:17
[2025-02-05T14:33:17.356+0000] {logging_mixin.py:188} INFO - 14:33:17  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-02-05T14:33:18.127+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v10.json is above dbt-ol supported version 7. This might cause errors.
[2025-02-05T14:33:18.178+0000] {local.py:494} INFO - Inlets: [Dataset(uri='postgres://postgres-db:5432/localiza_dw.dm_credit_operations.stg_credit_fraud', extra=None)]
[2025-02-05T14:33:18.179+0000] {local.py:495} INFO - Outlets: [Dataset(uri='postgres://postgres-db:5432/localiza_dw.dm_credit_operations.agg_locations_risk_score', extra=None)]
[2025-02-05T14:33:18.180+0000] {local.py:595} INFO - Assigning inlets/outlets without DatasetAlias
[2025-02-05T14:33:18.181+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-02-05T14:33:18.199+0000] {dag.py:3954} INFO - Setting next_dagrun for dag_dbt-model_credit_fraud to None, run_after=None
[2025-02-05T14:33:18.239+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-05T14:33:18.275+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_dbt-model_credit_fraud, task_id=model_credit_fraud.agg_locations_risk_score_run, run_id=manual__2025-02-04T00:00:00+00:00, execution_date=20250204T000000, start_date=20250205T143309, end_date=20250205T143318
[2025-02-05T14:33:18.341+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-05T14:33:18.362+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-05T14:33:18.364+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
